content/flux/v0.x/data-types/basic/float.md

```diff
@@ -2,11 +2,226 @@
+| 2021-01-01T00:00:00Z |                                              10.0 |
+| 2021-01-01T02:00:00Z |                                              20.0 |
+| 2021-01-01T03:00:00Z |                                              30.0 |
+| 2021-01-01T04:00:00Z |                                              40.0 |
+{{% /flex-content %}}
+{{< /flex >}}
+
+**To convert any column to floats**:
+
+1. Use [`map()`](/flux/v0.x/stdlib/universe/map/) to iterate over and rewrite rows.
+2. Use [`float()`](/flux/v0.x/stdlib/universe/float/) to convert columns values to floats.
+
+```js
+data
+  |> map(fn: (r) => ({ r with index: float(v: r.index) }))
+```
+
+{{< flex >}}
+{{% flex-content %}}
+##### Given the following input data:
+| \_time               | index _<span style="opacity:.5">(int)</span>_ |
+| :------------------- | --------------------------------------------: |
+| 2021-01-01T00:00:00Z |                                             1 |
+| 2021-01-01T02:00:00Z |                                             2 |
+| 2021-01-01T03:00:00Z |                                             3 |
+| 2021-01-01T04:00:00Z |                                             4 |
+{{% /flex-content %}}
+
+{{% flex-content %}}
+##### The example above returns:
+| \_time               | index _<span style="opacity:.5">(float)</span>_ |
+| :------------------- | ----------------------------------------------: |
+| 2021-01-01T00:00:00Z |                                             1.0 |
+| 2021-01-01T02:00:00Z |                                             2.0 |
+| 2021-01-01T03:00:00Z |                                             3.0 |
+| 2021-01-01T04:00:00Z |                                             4.0 |
+{{% /flex-content %}}
+{{< /flex >}}
+
+## Operate on floats
+
+- [Perform arithmetic operations on floats](#perform-arithmetic-operations-on-floats)
+- [Compare float values](#compare-float-values)
+- [Round float values](#round-float-values)
+- [Flux math package](#flux-math-package)
+
+### Perform arithmetic operations on floats
+To perform operations like adding, subtracting, multiplying, or dividing float values,
+use [Flux arithmetic operators](/flux/v0.x/spec/operators/#arithmetic-operators).
+Operands must be the same type.
+
+```js
+1.23 + 45.67
+// Returns 46.9
+
+1.23 - 45.67
+// Returns -44.440000000000005
+
+float(v: "12345e+67") * 100.0
+// Returns 1.2345000000000001e+73
+
+144.0 / 12.0
+// Returns 12.0
+
+10.0 ^ 2.0
+// Returns 100.0
+```
+
+{{% note %}}
+#### Inherent rounding errors in floating-point arithmetic
+To fit an infinite number of real values into a finite number of bits,
+computer systems must round floating-point values in arithmetic operations.
+This results in small rounding errors in some operations.
+{{% /note %}}
+
+### Compare float values
+Use [Flux comparison operators](/flux/v0.x/spec/operators/#comparison-operators)
+to compare float values.
+Operands must be the same type.
+
+```js
```

Comment:
~~~suggestion
Use [Flux comparison operators](/flux/v0.x/spec/operators/#comparison-operators)
to compare float values.
Operands must be the same type.
The operation returns a float.

```js
~~~